splinterd/src/config/builder.rs

```diff
@@ -69,6 +69,9 @@ impl ConfigBuilder {
         self
     }
 
+    /// In testing for the ConfigBuilder module behind the config-builder feature, this method is
+    /// not used in order to produce a None option in the resulting ConfigBuilder object.
+    #[allow(dead_code)]
```

Comment:
Should we guard these with a testing conditional instead of marking them with dead-code?

Also, these comments should probably not be doc strings.

Reply:
Addressed.